articles/windows-mdm-setup.md

```diff
@@ -0,0 +1,200 @@
+
+3. Register a test workstation with Autopilot
+
+4. Upload your organization's logo that end users will see during setup
+
+5. Test Autopilot
+
+### Step 1: buy a Microsoft Intune license
+
+Autopilot requires at least one Intune license to edit the Autopilot profile.
+
+1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com/).
+
+2. In the left-side bar select **Marketplace**.
+
+3. On the **Marketplace** page, select **All products** and in the search bar below **All products** enter "Intune".
+
+4. Find **Microsoft Intune Plan 1 Device** and select **Details**
+
+5. On the **Microsoft Intune Plan 1 Device** page, select **Buy** and follow instructions to purchase the license. 
+
+6. Sign in to [Entra admin center](https://entra.microsoft.com).
+
+7. In the left-side bar, select **Users > All users**.
+
+8. Select or create your Intune admin user and then select **Licenses**
+
+9. Select **+ Assignments** and assign the **Microsoft Intune Plan 1 Device** to this user.
+
+### Step 2: create an Autopilot profile
+
+1. Sign in to [Microsoft Intune](https://endpoint.microsoft.com/) using the Intune admin user from step 1.
+
+2. In the left-side bar select **Devices > Enroll devices**. Under **Windows Autopilot Deployment Program** select **Deployment Profiles** to navigate to the **Windows Autopilot deployment profiles** page.
+
+3. Select **+ Create profile > Windows PC** and follow steps to create an Autopilot profile. On the **Assignments** step, select **+ Add all devices**.
+
+### Step 3: register a test workstation
+
+1. Open your test workstation and follow these Microsoft instructions to export your workstations's device hash as a CSV. The CSV should look something like `DeviceHash_DESKTOP-2V08FUI.csv`
+
+2. In Intune, in the left-side bar, select **Devices > Enroll devices**. Under **Windows Autopilot Deployment Program** select **Devices** to navigate to the **Windows Autopilot devices** page.
+
+3. Select **Import** and import your CSV.
+
+4. After Intune finishes the import, refresh the **Windows Autopilot devices** page several times to confirm that your workstation is registered with Autopilot.
+
+### Step 4: upload your organization's logo
+
+1. Navigate to [Entra admin center](https://entra.microsoft.com).
+
+2. In the left-side bar select **Show more > User experiences > Company branding**.
+
+3. On the **Company Branding** page, select **Configure**.
+
+4. Under **Edit default sign-in experience** select the **Sign-in form** tab and upload your logo to the **Square logo (light theme)** and **Square logo (dark theme)** fields.
+
+5. In the bottom bar, select **Review + Save** and then **Save**.
+
+### Step 5: test Autopilot
+
+1. Wipe your test workstation.
+
+2. After it's been wiped, open your workstation and follow the setup steps. At screen in which you're asked to sign in, you should see the title "Welcome to [your organziation]!" next to the logo you uploaded in step 4.
```

Comment:
@noahtalerman FYI: I changed the placeholder text on this line (`<your organization>` » `[your organization]`) to prevent it from being interpreted as an HTML element when the page is built from Markdown.

Reply:
Thanks Eric!